packages/toml/index.js

```diff
@@ -0,0 +1,46 @@
+const { TomlDocument: TomlDocumentInner } = require("./pkg/covector_toml");
+
+function proxyPropGet(target, prop) {
+  return new Proxy(target.get(prop), {
+    get(t, innerProp) {
+      const ret = Reflect.get(...arguments);
+      return ret ? ret : proxyPropGet(target, prop + "." + innerProp);
+    },
+    set(t, innerProp, newValue) {
+      return target.set(prop + "." + innerProp, newValue);
+    },
+  });
+}
+
+class TomlDocument {
+  inner;
+
+  constructor(toml) {
```

Comment:
Would it be worth adding a `.parse()` method that calls the constructor and sets things up? It seems the `new` keyword is a bit out of vogue these days.

Reply:
yeah sure